spec/run_oc_pedant.rb

```diff
@@ -43,7 +43,15 @@
     '--skip-omnibus',
     '--skip-usags',
     '--exclude-internal-orgs',
-    '--skip-headers'
+    '--skip-headers',
+
+    # Chef 12 features not yet 100% supported by Chef Zero
```

Comment:
If we're going to accept this commit we definitely need to file another bug to go back and make these work so we get feature parity.  I don't know about `api-v1` but all the other features are new and just require adding.  Before we put out the next minor release I would like to make sure that chef-zero supports these.